test/sampling.jl

```diff
@@ -13,6 +13,7 @@ x = [randi(10) for i = 1:n]
 @test isa(x, Vector{Int})
 @test extrema(x) == (1, 10)
 @test isapprox(proportions(x, 1:10), fill(0.1, 10), atol=5.0e-3)
+@test randi(MersenneTwister(1), 1000) == randi(MersenneTwister(1), 1000)
```

Comment:
OK, got it. Maybe a comment would be useful? I'll leave it up to you whether it's worth adding the check using `Base.GLOBAL_RNG` too.